src/main/scala-2.10/sbtorgpolicies/settings/AllSettingsSpecific.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sbtorgpolicies.settings
+
+import sbt.Keys._
+import sbt._
+
+trait AllSettingsSpecific extends dependencies {
```

Comment:
Is there a naming convention for scala version specific files?

Reply:
Naming things is always the hardest part 😛 .

Maybe just `ScalaSettings`?

Reply:
More naming questions :)
How about `dependenciesSpecific.scala`? Or should I just merge both files under `ScalaSettings`?
https://github.com/47deg/sbt-org-policies/pull/510/files#diff-c498862754c166dea83d0b202b567241

Reply:
@juanpedromoreno bump

Reply:
We could merge it under ScalaSettings 👍